### ISSUE TITLE
Integrate the static gateway flow (Home Assistant ->  Apache reverse proxy server within ec2 instance -> User)

### ISSUE THREAD

**Issue by siddiqss** [oceanbuilders/smartpod#103]
SSH addon testing and integration with EC2 server

**Comment by amitpujar** [comment]
Need credentials to test. Can be done in Supervisor. Connect with Omer for more details - Omer to give permissions on 2nd December. Update on Tuesday, 6th Dec.

**Comment by omerll** [comment]
Waiting on asad @godslave05, will update on friday to amit

**Comment by godslave05** [comment]
work in progress
@godslave05  will try to reach @olagalal

**Comment by godslave05** [comment]
stuck , because of lack of testing environment, 
in current setup it is not possible to open port from Ha installed in Rpi located in Panama

**Comment by siddiqss** [comment]
On hold

**Comment by omerll** [comment]
On hold review later

**Comment by diegopau** [comment]
We are not using a solution based on SSH anymore. I am closing this for now. We can revisit this possibility in the future if needed.
